source/guides/maintenance/deploy-at-scale.rst

```diff
@@ -242,8 +242,8 @@ Continuously test its use; Automate its use by redeploying |CL| and
 application on new hosts. This naturally minimizes configuration drift,
 challenges your monitoring systems, and business continuity plans.
 
-.. _`Downloads`: https://clearlinux.org/downloads/
-.. _`Containers`: https://clearlinux.org/downloads/containers
+.. _`Downloads`: https://clearlinux.github.io/downloads/
+.. _`Containers`: https://clearlinux.github.io/downloads/containers
```

Comment:
I don't think this is supposed to be changed since it is going to the content part of the site. Anything under downloads probably needs to not change.